src/Atoms/Switch/Switch.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import Switch from './index'
+
+const StateHolder = ({ children = () => {} }) => {
+  const [checked, setChecked] = useState(false)
+
+  return <>
+    {checked ? 'Yup' : 'Nope'}
+    {children({ checked: checked, onChange: setChecked })}
```

Comment:
@monzey hinted me to use `React.cloneElement` instead of React Render Props (https://github.com/rgsystemes/lib-ui/blob/74c669c12c412cd68af422bb39d6bfb0cdc71b84/src/Molecules/UserSelect/UserSelect.test.jsx#L30)

Reply:
I do not understand what it is

Reply:
You got to replace the `children` call by `cloneElement` and remove the render props callback on your wrapper usages, it's just simplier